src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CodeActionEndpoint.cs

```diff
@@ -249,7 +249,21 @@ internal async Task<IEnumerable<RazorCodeAction>> GetCSharpCodeActionsFromLangua
                 return Array.Empty<RazorCodeAction>();
             }
 
-            var newRequest = context.Request with { Range = projectedRange };
+            var newContext = context.Request.Context;
+            if (context.Request.Context is OmniSharpVSCodeActionContext omniSharpContext &&
+                omniSharpContext.SelectionRange is not null &&
+                 _documentMappingService.TryMapToProjectedDocumentRange(
```

Comment:
The indentation of this line does not match the previous line <img width="18" alt="image" src="https://user-images.githubusercontent.com/754264/145150454-9257ecde-dadb-4bb7-8087-a469f730a0b5.png">

Reply:
lol did you paste a picture of that emoji

Reply:
I take my new employee hazing _very_ seriously.

Reply:
your hazing is both appreciated and makes me feel bad, which I think is the intent.